examples/webgl2_sandbox.html

```diff
@@ -34,17 +34,8 @@
 	<body>
 		<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl2 sandbox.</div>
 
-		<script type="module">
-
-			import { PerspectiveCamera } from '../src/cameras/PerspectiveCamera.js';
-			import { SphereBufferGeometry } from '../src/geometries/SphereGeometry.js';
-			import { MeshNormalMaterial } from '../src/materials/MeshNormalMaterial.js';
-			import { PointLight } from '../src/lights/PointLight.js';
-			import { Color } from '../src/math/Color.js';
-			import { Mesh } from '../src/objects/Mesh.js';
-			import { Fog } from '../src/scenes/Fog.js';
-			import { Scene } from '../src/scenes/Scene.js';
-			import { WebGL2Renderer } from '../src/renderers/WebGL2Renderer.js';
+		<script src="../build/three.js"></script>
```

Comment:
I would just leave the same `module` code as previously. If we're targeting a modern browser that support webgl2, it will support modules too and we could use it as an example of modern implementation.

Reply:
Very depends on the purpose of `webgl2_sandbox.html`. If it's just sandbox for WebGL 2.0, it's ok to replace. If it's also for `module` in addition to WebGL 2.0, we should leave it.

As of me, it's just sandbox so we can replace. And we can make another example for `module` if necessary.

Reply:
Yeah so maybe it makes sense to have another entry on the examples as `webgl_module` or so, to showcase the use of modules /cc @mrdoob

Reply:
I think the use of `type="module"` also means code can be modified from the source without having to rebundle/rebuild the library.

Reply:
Sorry for the delay. Yes, I would prefer to keep using modules in this file as it will allow us to iterate this new code faster.

Reply:
OK, I don't have strong opinion here.